examples/pointer-constraints.c

```diff
@@ -0,0 +1,150 @@
+#include <GLES2/gl2.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <wayland-client.h>
+#include <wayland-egl.h>
+#include <wlr/render/egl.h>
+#include "xdg-shell-client-protocol.h"
+#include "pointer-constraints-unstable-v1-client-protocol.h"
+
+#include <linux/input-event-codes.h>
```

Comment:
This is Linux-specific.

Reply:
it was copied from the idle inhibit example, but I'll look into it.